packages/core/test/utils.spec.js

```diff
@@ -299,8 +299,44 @@ describe('Utils', () => {
             expect(utils.relUrlPath('/path/to/image.png', '/path/b', opts)).toEqual('to/image.png');
         });
 
-        it('returns correct relative path to file with extension', () => {
-            expect(utils.relUrlPath('../to/image.png', '/path/b', opts)).toEqual('../to/image.png');
```

Comment:
Do not remove this test, as it was just added with another PR.

Reply:
Ack - the similar named tests confused me. Added it back, also added a variation with relativeToCurrentFolder being used.